ambari-server/src/main/java/org/apache/ambari/server/orm/entities/ServiceGroupEntity.java

```diff
@@ -70,6 +71,9 @@
   @Column(name = "service_group_name", nullable = false, insertable = true, updatable = true)
   private String serviceGroupName;
 
+  @Column(name = "stack_id", nullable = false, insertable = true, updatable = true)
+  private Long stackId;
+
```

Comment:
I think we need to make this an entity relationship instead since it's most likely that we'll need the entity data when dealing with service groups.

Reply:
this stackId is the foreign key of stack table. We can get stack entity from this key.

Reply:
That is true, but this means we'd need an extra step, like this:
```
@Inject StackDAO stackDAO;
stackDAO.findById(ServiceGroupEntity.getStackId())
```

vs just 
`ServiceGroupEntity.getStackEntity();`

I guess I'm OK leaving it, but why not change it now before consumers begin using it?